actions/register.ts

```diff
@@ -255,27 +276,48 @@ export class Registry {
     context: Context,
     network: string
   ): Promise<Registry> {
-    const str = await context.storage.getStr(storageKey(network));
+    const str = await context.storage.getStr(getOrdersStorageKey(network));
+    const lastNotifiedError = await context.storage
+      .getStr(LAST_NOTIFIED_ERROR_STORAGE_KEY)
+      .then((isoDate) => (isoDate ? new Date(isoDate) : null))
+      .catch(() => null);
```

Comment:
lastNotified is persisted so we avoid creating too many notifications in slack (not super sofisticated, but a small patch for the time being)